### DISCUSSION TITLE
Hii Guies

### DISCUSSION THREAD

**Discussion by Ganesh-Code-Live** [community/community#110297] [GitHub Education]
### Hi everyone,

My  name is ganesh and i am currently pursuing BCA ...Joined this github golab campus as it is my first time and i don't know anything about it ....It will be helpful if someone can give me tips related to it

**Comment by drdixit** [comment]
Just dropped here to say👋 hello @Ganesh-Code-Live 
It's a place where you meet new people and discover new technologies. You read others opinions and share your own. Sometimes it makes you happy, sometimes it doesn't. Try not to overthink it.
